### ISSUE TITLE
`uname` is failing on MacOS, causing `date` to also fail

### ISSUE THREAD

**Issue by gorthx** [hunleyd/todo#2]
Here's what happens:
```(main) :::-->todo list
todo: Listing existing todos:
uname: illegal option -- o
usage: uname [-amnprsv]
date: illegal option -- -
usage: date [-jnRu] [-r seconds|file] [-v[+|-]val[ymwdHMS]]
            [-I[date | hours | minutes | seconds]]
            [-f fmt date | [[[mm]dd]HH]MM[[cc]yy][.ss]] [+format]```

`uname`, `uname -s`, and `uname -v` all work;  I'll submit a patch.

**Comment by gorthx** [comment]
#3

**Comment by hunleyd** [comment]
fixed w/ #3 thx!
